### PR TITLE
chore(ci): exclude generated files from pr size calculations using globbing

### PR DESCRIPTION
**Pull request by czeslavo**
**What this PR does / why we need it**:

Excludes generated files from [pull-request-size](https://github.com/noqcks/pull-request-size) calculation. 

As it was discovered, the tool does not use `.gitattributes` globbing the way we would expect it to do (the same way as git does). Instead, it uses its own library for that purpose ([`minimatch`](https://github.com/noqcks/pull-request-size/blob/f839be1b2b91687584602f24e753325eeb0166c3/index.js#L99)). This change uses this intel to match all generated files in the repository using patterns that `minimatch` library will accept fine. 

It was tested against all generated files in the repository: https://codesandbox.io/s/js-playground-forked-fvx49r?file=/src/index.js

As I understand the tool code it uses [`.gitattributes` from main](https://github.com/noqcks/pull-request-size/blob/f839be1b2b91687584602f24e753325eeb0166c3/index.js#L68), so I expect this to take effect after merging. 

**PR Readiness Checklist**:

Complete these before marking the PR as `ready to review`:

- ~[ ] the `CHANGELOG.md` release notes have been updated to reflect any significant (and particularly user-facing) changes introduced by this PR~

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1 +1,3 @@
-zz_generated_*.go linguist-generated=true
+**/zz_generated*.go linguist-generated=true
+pkg/apis/** linguist-generated=true
```
Comment (pmalek) [inline .gitattributes:2]: This one I'd suggest to leave out since this is where the go structures are defined, by the developers.

```suggestion
```

**Review by pmalek — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 1 above)
Comment (czeslavo) [reply .gitattributes]: Good catch. There are only `zz_generated.deepcopy.go` which will get caught by the `zz_generated_*.go`.

**Review by pmalek — APPROVED** [review]
I'm fine with this. Please double check with @mflendrich as I believe we had a comment w.r.t last time.

**Comment by czeslavo** [comment]
> Please double check with @mflendrich as I believe we had a comment w.r.t last time.
We spoke about it yesterday and Michał suggested I should try if globbing would work. As it does, I think that's gonna be fine to use it. That's always less code to maintain.

**Pull request merged**